### PR TITLE
feat: Replace cli to cobra

### PR DESCRIPTION
**Pull request by anqiansong**

### CODE REVIEW

File: tools/goctl/cmd/root.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+	copyArgs := append([]string(nil), args...)
+	parentCmd, _, err := rootCmd.Traverse(args[:0])
+	if err != nil {
+		return copyArgs
+	}
+
+	for idx, arg := range copyArgs {
+		if idx == 0 {
+			continue
+		}
```
Comment (chenquan) [inline tools/goctl/cmd/root.go:51]:
```suggestion
	for idx, arg := range copyArgs[0:] {
```
Maybe it is more simple?

**Review by zhoushuguang — APPROVED** [review]
LGTM

**Review by kevwan — APPROVED** [review]
LGTM.

File: tools/goctl/cmd/root.go [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+		if assignIndex > 0 {
+			goStyleFlag += flagValue
+		}
+
+		copyArgs[idx] = goStyleFlag
+	}
+	return copyArgs
+}
+
+func init() {
```
Comment (kevwan) [inline tools/goctl/cmd/root.go:89]: Better to put init() at the beginning.

**Review by chenquan — APPROVED** [review]
LGTM

**Review by fynxiu — APPROVED** [review]
LGTM

**Pull request merged**